RocketChat/Rocket.Chat
# [NEW] Add the ability to set the character message limit on Livechat widget

## Proposed changes

Implementing a new character limit field in the LiveChat Aparence.

## Issue(s)

#17451

## How to test or reproduce

1. Open Livechat Aparence
2. Edit Characther Limit input
3. Save
4. reload the page and check if the information has been saved

If you want test with Livechat, you will need this [feature](https://github.com/RocketChat/Rocket.Chat.Livechat/pull/443)

## Screenshots

![image](https://user-images.githubusercontent.com/24482087/87487216-75210d00-c613-11ea-913a-68c5c023f636.png)

## Types of changes

## Checklist


## Changelog

additional setting in Admin panel for setting a character limit on livechat.

## Further comments
